src/modules/coreHttpApi/controllers/AccountController.ts

```diff
@@ -23,7 +24,8 @@ export class AccountController extends BaseController {
     @Accept("application/json")
     public async sync(): Promise<Envelope> {
         const result = await this.transportServices.account.syncEverything();
-        return this.ok(result);
+        if (result.isError) return this.ok(result);
```

Comment:
```suggestion
        if (result.isError) throw result.error;
```
this is what would happen deep in the this.ok call :)